charts/bandstand-web-service/templates/tests/pod.yaml

```diff
@@ -24,7 +24,7 @@ spec:
   {{- end }}
   containers:
     - name: {{ $relName }}-acceptance-tests
-      image: {{ .Values.dockerRegistry | default "ktechartifacts-docker.jfrog.io" }}/{{ $.Values.imageName | default $relName }}-acceptance-tests:{{ .Values.global.image.tag }}
+      image: {{ .Values.dockerRegistry | default "ktechartifacts-docker.jfrog.io" }}/{{ $.Values.test.imageName | default $.Values.imageName | default $relName }}-acceptance-tests:{{ .Values.global.image.tag }}
```

Comment:
Helm doesn't seem to like this syntax - any ideas @gavinclarkeuk ?

Reply:
Try this:
```suggestion
      image: {{ .Values.dockerRegistry | default "ktechartifacts-docker.jfrog.io" }}/{{ ($.Values.test).imageName | default $.Values.imageName | default $relName }}-acceptance-tests:{{ .Values.global.image.tag }}
```

Reply:
Things break if you have multi level keys and you don't define one of the higher levels. The extra brackets make it falsy if the interim key (test) doesn't exist